src/__tests__/TuxArchitectStores-test.js

```diff
@@ -139,9 +140,24 @@ describe('architect', function () {
     }).toThrow(new Error('Invariant Violation: store is waiting for a store that has not been registered to any actions.'));
   });
 
-  it('should throw an error if a store attempts to output the same string another store is already outputing', function () {
+  it('should allow multiple stores to output the same string', function () {
```

Comment:
Can remove this test, the test on line 150 will guarantee that this does not throw an error.